packages/react-components/review/beta/react-components.api.md

```diff
@@ -1934,8 +1934,8 @@ export const RTESendBox: (props: RTESendBoxProps) => JSX.Element;
 
 // @beta
 export interface RTESendBoxProps {
-    // (undocumented)
-    valueToDisplay?: string;
+    disabled?: boolean;
```

Comment:
Do we need to make `disabled` prop optional?

Reply:
![image](https://github.com/Azure/communication-ui-library/assets/73612854/6f3ea4e2-7f56-4d3e-8251-6528f53b1cf8)

We this marked as optional in the design doc. @AmyL219 was there a specific reason we wanted this as optional?

Reply:
It was based on the original SendBox public API design @palatter , @vhuseinova-msft